### ISSUE TITLE
connector-atelier-server image cannot be published

### ISSUE THREAD

**Issue by girarda** [airbytehq/airbyte#24920]
## Current Behavior
* The connector-atelier-server image cannot be published as part of the OSS release action because the [build args aren't passed to the docker build process](https://github.com/airbytehq/airbyte-platform-internal/blob/master/oss/tools/bin/publish_docker.sh#L53-L61).

Additionally, we're installing a lot of packages as part of the build, which adds to an already expensive release cycle. It would be nice to streamline it

## Proposed solution
- Hardcode the python version, or at least, set a default in the dockerfile
- Update the Dockerfile with the CDK version number as part of the `publish-cdk-command-manually` flow

Setting a default for these two variables will unblock the release.

## Alternative solutions
- Setting the CDK and Python versions from the [release_version script](https://github.com/airbytehq/airbyte-platform-internal/blob/master/oss/tools/bin/release_version.sh#L38). 
- The release version script would also need to be updated as part of `publish-cdk-command-manually`
- This is leaking details specific to the connector builder all the way to release scripts, which makes this solution less interesting

## Acceptance criteria
- [ ] The connector-builder-atelier is added to the projectDir list of the publish script
- [ ] OSS release succeeds in publishing the connector-builder-atelier image
- [ ] The airbyte-connector-builder-server-wrapped's Dockerfile is updated with the latest connector-atelier-server image that was published

**Comment by girarda** [comment]
grooming notes:
- alex to sync with catherine on what we want to do with the package installs / whether we should use pyenv
- additional acceptance criteria: update the versionbump config to also upgrade the atelier server to the latest version

**Comment by vrfn** [comment]
Please be aware that even though this image is currently not published due to the issue mentioned above, it *is* currently part of the default Helm chart installation.

I created an issue about a week ago but got no response: https://github.com/airbytehq/helm-charts/issues/12

**Comment by girarda** [comment]
Created separate issue for improving the build time https://github.com/airbytehq/airbyte/issues/25167
